src/main/java/teammates/common/datatransfer/questions/FeedbackMsqResponseDetails.java

```diff
@@ -35,7 +35,7 @@ public String getAnswerString() {
         boolean isOtherEnabled = msqQuestionDetails.isOtherEnabled();
 
         // number of Msq options selected including other option
-        int totalChoicesSelected = answers.size() + (isOther ? 1 : 0);
+        int totalChoicesSelected = answers.size();
```

Comment:
We might want to cover the case with a test case.

Reply:
Leave it to #1501